script/csharp/compile-command.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# This script does two things
+#   - Redirects errors from stdout to stderr so that nztrain knows when compilation fails
+#   - Supply .dll references to the csc command
+
+# Print version and copyright by compiling nothing
+/usr/bin/dotnet /usr/share/dotnet/sdk/3.1.101/Roslyn/bincore/csc.dll | \
+# print the first line, which has compiler version info, to stdout
+head -n 1 >&1
+
+# Actually compile:
+# [dotnet] [csc] -reference:[stdlib.dll] [...passed in args...]
+/usr/bin/dotnet /usr/share/dotnet/sdk/3.1.101/Roslyn/bincore/csc.dll \
+-reference:/usr/share/dotnet/sdk/3.1.101/ref/netstandard.dll "$@" | \
+# Remove the first 3 lines of output. The rest, if any, are errors.
+tail -n +4 >&2
```

Comment:
```suggestion
tail -n +4 1>&2

exit ${PIPESTATUS[0]}
```
- To be consistent with the other compiler commands in `languages.yml`.
- `exit` with the status code of the compilation so the judge knows if the compilation failed.

Reply:
Ah, this was the bit that I was missing. I erroneously figured that nztrain was using the stderr channel to determine if the compilation succeeded or failed. Great work, thanks!

Since this is the case, there is now no need to run the compiler command twice. I'll resolve these two requests with a single commit.